fe1-web/src/features/social/components/TextInputChirp.tsx

```diff
@@ -94,19 +98,22 @@ const TextInputChirp = (props: IPropTypes) => {
           onFocus={() => setFocused(true)}
           onBlur={() => setFocused(false)}
         />
-        <View style={styles.buttonView}>
-          <View style={styles.charsLeft}>
-            <Text style={textIsRed ? [Typography.base, Typography.error] : [Typography.base]}>
-              {charsLeft.toString()}
-            </Text>
+        <View style={styles.bottomRow}>
+          {errorMessage && <Text style={[Typography.base, Typography.error]}>{errorMessage}</Text>}
```

Comment:
Is this && really needed? I mean, if the next is undefined, it will not show anything anyway, right?

Reply:
Won't the Text component still be built, resulting in empty divs ?